armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -630,7 +630,7 @@ def test_updateComponentTempsBy1DTempFieldRuntimeError(self):
     def test_AssemblyAxialExpansionException(self):
         """test that negative height exception is caught"""
         # manually set axial exp target component for code coverage
-        self.a[0].axialExpTargetComponent = self.a[0][0]
+        self.a[0].p.axialExpTargetComponent = self.a[0][0]
```

Comment:
Should this be `self.a[0][0].name`?

The parameter is supposed to be a string, but since this test is intended to raise an error I can't tell if this assignment is _supposed_ to be invalid or not.

Reply:
Yep! You're 100% right. 
https://github.com/terrapower/armi/pull/1256/commits/ab51357d6e88166e691505ac91ed97a536c9d417